packages/localize/src/formatters/xliff.ts

```diff
@@ -250,9 +250,10 @@ export class XliffFormatter implements Formatter {
 
       if (descStack.length > 0) {
         // https://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#note
-        const note = document.createElement('note');
+        const note = doc.createElement('note');
```

Comment:
ouch!

Can you remove the `"DOM"` lib from the tsconfig? Hopefully TS would catch this.

Reply:
Done. (But DOM types still get loaded because the `xmldob` package I'm loading has a reference to them)

Reply:
Filed an issue here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/49953